src/Storage/ClaimController.ts

```diff
@@ -79,13 +80,22 @@ export class ClaimController {
     retryDelay?: number
     maxAttempts?: number
   } = {}) {
-    return asyncPipe(
-      this.findEntryToDownload,
-      this.updateEntryAttempts,
-      this.downloadEntryClaim,
-      this.updateEntryPairs,
-      this.publishEntryDownload
-    )({ retryDelay, maxAttempts })
+    this.logger.child({ method: 'downloadNextHash' })
+    try {
+      this.logger.info('Downloading next entry')
+      const result = await asyncPipe(
+        this.findEntryToDownload,
+        this.updateEntryAttempts,
+        this.downloadEntryClaim,
+        this.updateEntryPairs,
+        this.publishEntryDownload
+      )({ retryDelay, maxAttempts })
+      this.logger.info(result, 'Successfully downloaded entry')
+      return result
+    } catch (error) {
+      if (error instanceof NoMoreEntriesException) return this.logger.info(error.message)
```

Comment:
I think we can avoid logging this one altogether, since it's part of the normal flow of things. What to you think?

Reply:
Yup I agree